src/sentry/api/serializers/models/group.py

```diff
@@ -824,7 +823,9 @@ def get_attrs(self, item_list, user):
                 )
             for item in item_list:
                 if self.has_dynamic_issue_counts:
-                    attrs[item].update({"filtered_stats": filtered_stats[item.id]})
+                    attrs[item].update(
+                        {"filtered_stats": filtered_stats[item.id] if filtered_stats else {}}
```

Comment:
I think this should still be `"stats"` instead of `"filtered_stats"`

Reply:
On line 844 "filtered_stats" is renamed to stats when brought into the "filtered" dict:
`attrs["filtered"].update({"stats": {self.stats_period: attrs["filtered_stats"]}})`